google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudType.java

```diff
@@ -148,6 +159,36 @@ public ManagedVmDeploymentConfiguration createDefaultConfiguration(
           new CloudSdkAppEngineHelper(new File(server.getConfiguration().getCloudSdkExecutablePath()),
               server.getConfiguration().getCloudProjectName()));
     }
+
+    protected static class PlaceholderDeploymentSource extends ModuleDeploymentSourceImpl {
```

Comment:
In my current iteration (still not pushed) I had updated this to FileSystemDeploymentSource. 

But I like your wording better.